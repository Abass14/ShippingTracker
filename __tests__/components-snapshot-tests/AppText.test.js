import ShallowRenderer from 'react-shallow-renderer';
import AppText from '../../src/app/components/text';

it('should render AppText correctly', () => {
    const renderer = new ShallowRenderer()
    const tree = renderer.render(
        <AppText />
    )
    expect(tree).toMatchSnapshot()
})