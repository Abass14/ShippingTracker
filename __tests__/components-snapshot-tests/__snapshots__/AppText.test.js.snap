// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AppText correctly 1`] = `
<Text
  style={
    [
      {
        "fontFamily": "Montserrat-Regular",
      },
      {
        "color": "#000",
      },
      undefined,
    ]
  }
/>
`;
